types/api-schema.ts

```diff
@@ -0,0 +1,7 @@
+export type TUser = {
```

Comment:
I'm curious, so far I only found this being used in `volunteer.js` file, is there another reason for this type being extracted into its own file or am I missing something?

Reply:
Yes, User exists other places too. I only have converted one File with User